platform/net/zephyr/src/mender-http.c

```diff
@@ -257,6 +371,26 @@ mender_http_response_cb(struct http_response *response, enum http_final_call fin
     }
 }
 
+static void
+artifact_response_cb(struct http_response *response, MENDER_ARG_UNUSED enum http_final_call final_call, void *user_data) {
```

Comment:
Pre-existing,

Please add `MENDER_ARG_UNUSED enum...` to `mender_http_response_cb` also.